packages/repository/package.json

```diff
@@ -26,8 +26,8 @@
     "@loopback/testlab": "^0.1.0"
   },
   "dependencies": {
-    "@loopback/context": "^0.1.0",
-    "loopback-datasource-juggler": "^3.9.2"
+    "@loopback/context": "^4.0.0-alpha.31",
```

Comment:
We've recently changed our versioning format, so please revert the change made here.